calico/getting-started/kubernetes/requirements.mdx

```diff
@@ -2,12 +2,78 @@
 description: Review requirements before installing Calico to ensure success.
 ---
 
-import ReqsSys from '@site/calico/_includes/components/ReqsSys';
-import ReqsKernel from '@site/calico/_includes/components/ReqsKernel';
-
 # System requirements
 
-<ReqsSys orch='Kubernetes' />
+## Node requirements
+
+* x86-64, arm64, ppc64le, or s390x processor
+
+* Calico must be able to manage `cali*` interfaces on the host.
+  When IPIP is enabled (the default), Calico also needs to be able to manage `tunl*` interfaces.
+  When VXLAN is enabled, Calico also needs to be able to manage the `vxlan.calico` interface.
+
+* Linux kernel 3.10 or later with [required dependencies](#kernel-dependencies). The following distributions have the required kernel, its dependencies, and are known to work well with Calico and Kubernetes.
+  * RedHat Linux 7
+  * CentOS 7
+  * Flatcar Container Linux
+  * Fedora CoreOS
+  * Ubuntu 18.04
+  * Debian 8
```

Comment:
We need to update this minimum requirement list as most distros are eol'ed and kernel 3.x maybe too old for some Calico feature to work.

Reply:
Yes, there are A LOT of updates that need to happen. This PR is the first step on a path to overhauling the system requirements.

I'll be in touch to talk specifics in the coming weeks.

Reply:
(Here, I'm just changing the way we build the page. No substantive changes to the content.)